bin/manual-mapfile-from-gumsdb.py

```diff
@@ -101,7 +101,7 @@ def get_vug_pattern(vug):
 
 print "Writing %s ..." % GRID_MAPFILE
 with open(GRID_MAPFILE, 'w') as f:
-    print >>f, "# /etc/grid-security/%s" % GRID_MAPFILE
+    print >>f, "# /etc/grid-security/grid-mapfile"
```

Comment:
Did you forget the "Add to" comment like you added for BAN_MAPFILE above?

Reply:
Derp, yes.